### ISSUE TITLE
mixed citation style

### ISSUE THREAD

**Issue by knarrff** [the-teachingRSE-project/competencies#301]
We sometimes add author names in the text when referring to citations, and sometimes not. I feel we should do one or the other, not both - except in very special circumstances where there is a clear reason why the authors are mentioned in the text.

**Comment by knarrff** [comment]
fixed with c227442

**Comment by MakisH** [comment]
I think at some point there was a discussion on whether citation references should act as words in a sentence. Compare:

```diff
- [1] showed that the Earth is round.
+ Foomann et al. [1] showed that the Earth is round.
```

I prefer the second version, thinking always that citation marks can be superscripts and, hence, visually destroy the sentence.

But let's delegate this to the journal editing step. #219
